src/OrleansServiceBus/Providers/Streams/EventHub/EventHubAdapterReceiver.cs

```diff
@@ -224,30 +246,56 @@ public Task MessagesDeliveredAsync(IList<IBatchContainer> messages)
             }
         }
 
-        private static async Task<EventHubReceiver> CreateReceiver(EventHubPartitionSettings partitionSettings, string offset, Logger logger)
+#if NETSTANDARD
+        private static async Task<PartitionReceiver> CreateReceiver(EventHubPartitionSettings partitionSettings, string offset, Logger logger)
+#else
+        private static async Task<EventHubReceiver> CreateReceiver(EventHubPartitionSettings partitionSettings, string offset, Logger logger) 
+#endif
         {
             bool offsetInclusive = true;
-            EventHubClient client = EventHubClient.CreateFromConnectionString(partitionSettings.Hub.ConnectionString, partitionSettings.Hub.Path);
+#if NETSTANDARD
+            var connectionStringBuilder = new EventHubsConnectionStringBuilder(partitionSettings.Hub.ConnectionString)
+            {
+                EntityPath = partitionSettings.Hub.Path
+            };
+            EventHubClient client = EventHubClient.CreateFromConnectionString(connectionStringBuilder.ToString());
+#else
+            EventHubClient client = EventHubClient.CreateFromConnectionString(partitionSettings.Hub.ConnectionString, partitionSettings.Hub.Path); 
+
             EventHubConsumerGroup consumerGroup = client.GetConsumerGroup(partitionSettings.Hub.ConsumerGroup);
             if (partitionSettings.Hub.PrefetchCount.HasValue)
             {
                 consumerGroup.PrefetchCount = partitionSettings.Hub.PrefetchCount.Value;
             }
+#endif
             // if we have a starting offset or if we're not configured to start reading from utc now, read from offset
-            if (!partitionSettings.Hub.StartFromNow || offset != EventHubConsumerGroup.StartOfStream)
+            if (!partitionSettings.Hub.StartFromNow ||
+#if NETSTANDARD
+                offset != PartitionReceiver.StartOfStream)
```

Comment:
https://github.com/Azure/azure-event-hubs-dotnet/blob/8774bc4afbfbe44f1722ee648cbaca7ae07edede/src/Microsoft.Azure.EventHubs/PartitionReceiver.cs#L28

Yes it is. I confirmed before.